x-pack/plugins/security_solution/public/threat_intelligence/routes.tsx

```diff
@@ -31,6 +31,7 @@ const ThreatIntelligence = memo(() => {
   const securitySolutionContext: ThreatIntelligenceSecuritySolutionContext = {
     getFiltersGlobalComponent: () => FiltersGlobal,
     licenseService,
+    timeLineService: timelines,
```

Comment:
so it would not work without the context, right?

Reply:
hmmmm thanks for bringing this up, it seems I can actually get it from kibana services directly, let me double check and clean up the code it that is the case. I don't think I need to add this context logic...

Reply:
you were right, it can totally be done without the context. I'm using kibana services directly now!